test/fixtures/dialects/tsql/insert_statement.sql

```diff
@@ -28,3 +28,11 @@ SELECT
 	[SOURCE]
 FROM 
    STAGE.ECDC_CASES
+GO
+
+BEGIN
+  INSERT INTO HumanResources.NewEmployee   
+      SELECT EmpID, LastName, FirstName, Phone,   
+             Address, City, StateProvince, PostalCode, CurrentFlag  
+      FROM EmployeeTemp;  
+END
```

Comment:
could we add another test case without the BEGIN END block as well?

```sql
INSERT INTO HumanResources.NewEmployee   
    SELECT
        EmpID,
        LastName,
        FirstName,
        Phone,   
        Address, City,
        StateProvince,
        PostalCode,
        CurrentFlag  
    FROM EmployeeTemp;
```

Reply:
Done!